src/component/mxgraph/shape/render/IconPainter.ts

```diff
@@ -352,23 +354,19 @@ export default class IconPainter {
    * This icon is used by `exclusive gateway`.
    */
   public paintXCrossIcon(paintParameter: PaintParameter): void {
-    const canvas = this.newBpmnCanvas(paintParameter, { height: 0.5, width: 0.5 });
-
-    IconPainter.drawCrossIcon(canvas);
-    const rotationCenterX = paintParameter.shape.w * paintParameter.ratioFromParent;
-    const rotationCenterY = paintParameter.shape.h * paintParameter.ratioFromParent;
+    const canvas = this.drawCrossIcon(paintParameter);
+    // only work for centered cross
```

Comment:
Normally, the IconPainter should ignore / be independent of the position of the icon in the shape.

Reply:
done with 5c90b55